### PR TITLE
Trial using pull request - 7

### PR DESCRIPTION
**Pull request by exile2003**

### CODE REVIEW

**Comment by Vitalii-alterego** [comment]
This doesn,t have point.

File: src/index.js [hunk 1]
```diff
@@ -9,7 +9,6 @@ import FileSaver from 'file-saver';
 //import '../node_modules/jquery/dist/jquery.min.js';
 //import './js/bootstrap-filestyle.min.js';
 
-
```
Comment (Vitalii-alterego) [inline src/index.js:12]: Why you added this?

**Comment by exile2003** [comment]
Yes. It's done for check.

**Comment by Vitalii-alterego** [comment]
Some more useless comment)

File: src/index.js (hunk 1 above)
Comment (exile2003) [reply src/index.js:12]: Just for check

File: src/index.js [hunk 2]
```diff
@@ -9,7 +9,6 @@ import FileSaver from 'file-saver';
...
 //import '../node_modules/bootstrap/dist/css/bootstrap.min.css';
```
Comment (Vitalii-alterego) [inline src/index.js:12]: Why you placed comment here

Comment (exile2003) [reply src/index.js:12]: I don't now(

**Pull request merged**